### PR TITLE
Improving OCP.create() to support creation of resource name

### PR DESCRIPTION
**Pull request by RazTamir**

### CODE REVIEW

File: ocs/ocp.py [hunk 1]
```diff
@@ -87,20 +87,31 @@ def get(self, resource_name='', out_yaml_format=True, selector=None):
...
+            raise CommandFailed(
+                "At least one of resource_name or yaml_file have to "
+                "be provided"
+            )
+        command = "create "
+        if yaml_file:
+            command += f"-f {yaml_file}"
+        elif resource_name:
+            # e.g "oc namespace my-project"
+            command = f"{self.kind} {resource_name}"
```
Comment (ebenahar) [inline ocs/ocp.py]: not sure if 'project' is considered as a kind here

Comment (RazTamir) [reply ocs/ocp.py]: It is not.
That's why added example line before:
# e.g "oc namespace my-project"
namespace and project are same

Comment (ebenahar) [reply ocs/ocp.py]: @RazTamir +1

File: ocs/ocp.py [hunk 2]
```diff
@@ -152,6 +163,21 @@ def apply(self, yaml_file):
         command = f"apply -f {yaml_file}"
```
Comment (keesturam) [inline ocs/ocp.py]: Did you miss to delete this line?

Comment (RazTamir) [reply ocs/ocp.py]: no, why?
This is the command for apply

Comment (keesturam) [reply ocs/ocp.py]: if my understanding is correct, this nullifies whatever we set for  "command" in the previous lines.

Comment (keesturam) [reply ocs/ocp.py]: the misunderstanding was due to the fact that there were hidden lines of code :-) Thanks @RazTamir for pointing out that! LGTM.

**Pull request merged**